templates/terraform/update_encoder/reservation.go.erb

```diff
@@ -0,0 +1,18 @@
+<%# The license inside this block applies to this file.
+	# Copyright 2020 Google Inc.
+	# Licensed under the Apache License, Version 2.0 (the "License");
+	# you may not use this file except in compliance with the License.
+	# You may obtain a copy of the License at
+	#
+	#     http://www.apache.org/licenses/LICENSE-2.0
+	#
+	# Unless required by applicable law or agreed to in writing, software
+	# distributed under the License is distributed on an "AS IS" BASIS,
+	# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	# See the License for the specific language governing permissions and
+	# limitations under the License.
+-%>
+obj["specificSkuCount"] = obj["specificReservation"].(map[string]interface{})["count"]
```

Comment:
Since the API request body is only a single field ("specificSkuCount") is there a reason to modify obj and delete "specificReservation" or can we just return a brand new object containing the sku count? I know there aren't any other update methods on this resource yet but if there were it might be safer to not modify the existing obj if it's not needed.